contrib/src/main/java/com/datatorrent/contrib/cassandra/AbstractCassandraInputOperator.java

```diff
@@ -45,7 +47,8 @@
 public abstract class AbstractCassandraInputOperator<T> extends AbstractStoreInputOperator<T, CassandraStore> {
 
   private static final Logger logger = LoggerFactory.getLogger(AbstractCassandraInputOperator.class);
-
+  private PagingState nextPageState;
+  private int fetchSize;
   int waitForDataTimeout = 100;
   @AutoMetric
   protected long tuplesRead;
```

Comment:
One of the usecase I have now is show right count on UI. There could be other usecases where people need the right count of emitted tuples.